config/default/views.view.localgov_events_listing.yml

```diff
@@ -1456,7 +1456,7 @@ display:
     display_options:
       exposed_block: true
       display_extenders: {  }
-      path: events
+      path: children-young-people-and-families/fostering/events
```

Comment:
Should this be a new view or is it ok for the /events page to be removed?

Reply:
This is okay for now. We'll figure it out properly soon (we'll probably take the same approach as news - newsrooms and articles/eventsroom and events)